lib/events/firestoreevents/firestoreevents.go

```diff
@@ -630,7 +630,7 @@ func (l *Log) searchEventsOnce(fromUTC, toUTC time.Time, namespace string, limit
 func (l *Log) SearchSessionEvents(fromUTC, toUTC time.Time, limit int, order types.EventOrder, startKey string, cond *types.WhereExpr) ([]apievents.AuditEvent, string, error) {
 	filter := searchEventsFilter{eventTypes: []string{events.SessionEndEvent}}
 	if cond != nil {
-		condFn, err := events.ToEventFieldsCondition(cond)
+		condFn, err := utils.ToFieldsCondition(cond)
```

Comment:
```golang
SearchSessionEvents(fromUTC, toUTC time.Time, limit int, order types.EventOrder, startKey string, cond *types.WhereExpr)
```

Another candidate to refactor method parameters to a request object

Reply:
Yep, it's a mouthful.

I'll take a look, but I won't make promises for this particular one - idk how much trouble it might be to change it and it's not strictly part of the PR.

Reply:
Added a TODO to follow up in a separate PR - I'd like to keep the scope here somewhat contained. If folks like the final result of GetSessions then we can go for something similar here.